aether-ui/aether/ui/assets/apps/pipeline/Pipeline.jsx

```diff
@@ -294,6 +324,126 @@ class Pipeline extends Component {
     )
   }
 
+  renderDeletionModal () {
```

Comment:
_one component to rule them all_ :sob: 

Would it be hard to refactor all of this to a new `Component`? :grimacing:

Reply:
😓